jvm/src/main/scala/com/intel/oap/expression/AggregateFunctionsBuilder.scala

```diff
@@ -17,19 +17,20 @@
 
 package com.intel.oap.expression
 
+import com.intel.oap.expression.ConverterUtils.FunctionConfig
 import com.intel.oap.substrait.expression.ExpressionBuilder
-
 import org.apache.spark.sql.catalyst.expressions.aggregate._
 
 object AggregateFunctionsBuilder {
 
   def create(args: java.lang.Object, aggregateFunc: AggregateFunction): Long = {
     val functionMap = args.asInstanceOf[java.util.HashMap[String, java.lang.Long]]
     aggregateFunc match {
-      case sum: Sum => {
-        ExpressionBuilder.newScalarFunction(functionMap, "SUM")
-      }
-      case other => null
+      case sum: Sum =>
+        val functionName = ConverterUtils.makeFuncName(
+          "sum", Seq(sum.child.dataType), FunctionConfig.OPT)
```

Comment:
suggestion: All the function names are defined as a final static constants.

Reply:
fixed in the latest commit.